app/src/main/res/layout/activity_detail.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
```

Comment:
here is better to use Constraint layout because we have views which are connected to each other.